src/components/shoppingCart/cartdetail/CartDetail.jsx

```diff
@@ -67,11 +119,27 @@ const CartDetail = () => {
         </Col>
       </Col>
       <Col className="productQuantityCol productColItem">
-        <Button className="countModifier">-</Button>
-        <span className="countNumber">1</span>
-        <Button className="countModifier">+</Button>
+        <Button
+          className="countModifier"
+          onClick={() => {
+            dispatch(decreaseProductQuantity(productsData));
+            dispatch(removeOneMoreItem(allItemsTotalPrice, props.info.price));
+          }}
+        >
+          -
+        </Button>
+        <span className="countNumber">{props.info.quantity}</span>
+        <Button
+          className="countModifier"
+          onClick={() => {
+            dispatch(increaseProductQuantity(productsData));
+            dispatch(addOneMoreItem(allItemsTotalPrice, props.info.price));
```

Comment:
so i feel like the cart structure is a bit off. addonemoreitem and increaseproductquantity should be the exact same thing unless i'm misunderstanding?